packages/passwordless-auth-construct/src/types.ts

```diff
@@ -342,3 +345,38 @@ export type KmsConfig = {
 export enum PasswordlessErrorCodes {
   CODE_MISMATCH_EXCEPTION = 'CodeMismatchException',
 }
+
+export type CreateUserParams = {
+  userPoolId: string;
+  username: string;
+  // this is the format for Cognito UserPool API, eslint not happy otherwise
+  // eslint-disable-next-line
```

Comment:
Our construct doesn't have to use the same prop names as the Cognito UserPool API. We can translate between `phoneNumber` and `phone_number` when making service calls